aii-pxelinux/src/main/pan/quattor/aii/pxelinux/schema.pan

```diff
@@ -11,8 +11,14 @@ include 'pan/types';
     PXE configuration
 }
 type structure_pxelinux_pxe_info = {
-    "initrd" : string
+    @{Kernel path (string in exact syntax).
+      If this contains a '@pattern@' substring, the kernel path is generated based on
+        the (first) enabled SPMA repository with name matching this glob pattern (without the '@').}
```

Comment:
Is this comment correct? How do the `kernel` and `initrd` paths related to SPMA repository names?

Reply:
```
          "initrd": "@rhel_kickstart@/images/pxeboot/initrd.img",
          "kernel": "@rhel_kickstart@/images/pxeboot/vmlinuz",
```

with eg https://download.rockylinux.org/pub/rocky/8/BaseOS/x86_64/kickstart/ the (disabled) kickstart repo.

Reply:
Aha, I see, thanks!